recipes/turbo/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "turbo" %}
+{% set version = "v0.6.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/gottingen/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: eda48c6fea7286349cb7484b04cdf66a5583aac31cf5d4f25bee8192970936ec
+
+build:
+  number: 0
+  skip: True  # [win]
```

Comment:
What's the reason osx and win are skipped? The build files look OK.

Reply:
I mainly testing it on linux, it is a server side library, I am not take too many work on that platform, may not test enough, so I igore it for a time, when so time needed, I'd like add it.

Reply:
for example ,I am add a  fiber feature now for the lib, I known how to deal it a linux platform, but on mac M1 or windows, I am not very sure about it